vspace/src/main/java/edu/asu/diging/vspace/web/staff/ExhibitionConfigurationController.java

```diff
@@ -0,0 +1,52 @@
+package edu.asu.diging.vspace.web.staff;
+
+import java.io.IOException;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import edu.asu.diging.vspace.core.data.DefaultExhibitionRepository;
+import edu.asu.diging.vspace.core.data.SpaceRepository;
+import edu.asu.diging.vspace.core.model.impl.DefaultExhibition;
+import edu.asu.diging.vspace.core.model.impl.Space;
+import edu.asu.diging.vspace.core.services.impl.DefaultExhibitionManager;
+import edu.asu.diging.vspace.core.services.impl.SpaceManager;
+
+@Controller
+public class ExhibitionConfigurationController {
+
+	@Autowired
+	private SpaceRepository spaceRepo;
+	@Autowired
+	private SpaceManager spaceManager;
+	@Autowired
+	private DefaultExhibitionManager exhibitManager;
+	@Autowired
+	private DefaultExhibitionRepository exhibitRepo;
+	
+	@RequestMapping("/staff/exhibit/econfig")
```

Comment:
since you use /staff/exhibit/config/add below, this should be /staff/exhibit/config